open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -131,23 +144,79 @@ def opensea_traits_to_token_metadata(asset_traits: list) -> TokenMetadata:
     )
 
 
-def get_collection_with_metadata(
+def get_tokens_from_opensea(
+    opensea_slug: str, token_ids: list[int]
+) -> list[Token]:
+    """Fetches eth nft data from opensea API and stores them into Token objects
```

Comment:
This is not numpy style pydoc string . Could you pls format it with this tool hosted on VSCode https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring ?

Reply:
That is what i use but it defaults to google docstring format - are we going for numpy for sure?

Reply:
Updated